src/java/picard/analysis/CollectJumpingLibraryMetrics.java

```diff
@@ -55,13 +55,36 @@
  * @author ktibbett@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = "Computes jumping library metrics.  Gets all data for computation from the first" +
-                "read in each pair and assumes that the MQ tag is set with the mate's mapping quality.  If the " +
-                "MQ tag is not set, then the program assumes that the mate's mapping quality is >= MINIMUM_MAPPING_QUALITY",
-        usageShort = "Produces jumping library metrics for the provided SAM/BAMs",
+        usage = CollectJumpingLibraryMetrics.USAGE_SUMMARY + CollectJumpingLibraryMetrics.USAGE_DETAILS,
+        usageShort = CollectJumpingLibraryMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
+
 public class CollectJumpingLibraryMetrics extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "SAM file generated with a jumping library, produced by the " +

```

Comment:
Yes.  There is a additional text that was never added for some reason.  Thanks for pointing this out!